pennylane/_device.py

```diff
@@ -422,4 +414,3 @@ def reset(self):
         After the reset the backend should be as if it was just constructed.
         Most importantly the quantum state is reset to its initial value.
         """
-        raise NotImplementedError
```

Comment:
Same here, `var` is not abstract, so should keep the `NotImplementedError`

Reply:
This way, if a device does not support variance, but the user calls `return var()`, they will get an error, rather than nothing happening :slightly_smiling_face: